### PR TITLE
:space_invader: lots of changes in css

### PR DESCRIPTION
**Pull request by mgiraldo**
- added typekit
- moved css from about to main css
- changed about layout
- added nypl logo
- change "see back photo" link to include nypl logo (and tried to point to DC link in JS but didnt work... see js)
- removed "more-on-back" link and replaced with "nypl-link" (see above)

### CODE REVIEW

**Comment by danvk** [comment]
Can you include a screenshot of your changes?

**Comment by mgiraldo** [comment]
oldnyc - historic photographs of new york city-1
oldnyc - historic photographs of new york city-2

about oldnyc

about oldnyc-1

**Comment by danvk** [comment]
I'm not sure about that font—is it a comic font?

**Comment by mgiraldo** [comment]
https://typekit.com/fonts/rooney-sans

**Comment by danvk** [comment]
Closed in favor of #86.

Let me know if including the NYPL logo is a hard requirement. I'd probably put it in the lower left of the page if so.

**Comment by mgiraldo** [comment]
it is

**Comment by mgiraldo** [comment]
this request also includes changes to the about page and the header. are you including those in #86?

**Comment by danvk** [comment]
I've added the logo in a [separate PR][1].

re: header, are you referring to the tagline? ("Mapping historical photos from the NYPL"?) I talked with Shana about tightening this up and she was fine with the shorter version. I don't see any reason to go back to the longer tagline.

re: about page, I think you're right about just having it open in a new tab. I misunderstood the NYPL request to make it "feel like part of the site". Can you summarize what your changes were other than that? It looks like you changed the font, made the links read and made the body much narrower?

[1]: https://github.com/oldnyc/oldnyc.github.io/pull/1

**Comment by mgiraldo** [comment]
re about:

you can see the changes here https://github.com/NYPL/oldnyc/commit/31e9360ee4e21d25b000021a2db9579c79a63cac

- mga

**Comment by danvk** [comment]
is my summary accurate?

**Comment by mgiraldo** [comment]
among the changes:

1. floated images to the right, added more margins and made them same width and appearance
2. removed link in header
3. added a link to the main page below header
4. changed the typeface
5. made ordered lists design consistent with unordered ones
6. fixed heading hierarchy
7. made text column width more readable
8. layout responds well to different window widths
